platform-workflow-module/src/main/java/com/softicar/platform/workflow/module/WorkflowI18n.java

```diff
@@ -122,6 +124,8 @@ public interface WorkflowI18n extends CoreI18n {
 		.de("Offene Aufgaben");
 	I18n2 PERMISSION_ARG1_OF_WORKFLOW_TRANSITION_ARG2_IS_MISSING = new I18n2("Permission '%s' of workflow transition '%s' is missing.")//
 		.de("Berechtigung %s des Arbeitsablauf-Übergangs %s fehlt.");
+	I18n0 PLEASE_PROVIDE_A_RATIONALE = new I18n0("Please provide a rationale.")//
```

Comment:
"comment" vs "rationale" - should we decide for one of those terms? (seem to refer to the same thing, in this context)
Probably okay as-is though.